shuup/simple_cms/models.py

```diff
@@ -144,7 +156,7 @@ class Page(MPTTModel, TranslatableModel):
         "Check this if this page should have a visible title"
     ))
 
-    objects = TreeManager.from_queryset(PageQuerySet)()
+    objects = PageQueryManager()
```

Comment:
We adding this delete view requires this change?

Reply:
reverted